package.json

```diff
@@ -1,6 +1,6 @@
 {
     "private": true,
-    "name": "react-admin",
+    "name": "react-admin-lerna",
```

Comment:
??

Reply:
Because jest was complaining about two packages having the same name

Reply:
Besides, this package.json will never be published on npm so it doesn't really matter right ?

Reply:
ok